marklogic-unit-test-client/src/main/java/com/marklogic/test/unit/JaxpServiceResponseUnmarshaller.java

```diff
@@ -67,23 +67,32 @@ public TestSuiteResult parseTestSuiteResult(String xml) {
 		for (int i = 0; i < tests.getLength(); i++) {
 			Element testNode = (Element) tests.item(i);
 			String testName = testNode.getAttribute("name");
+
 			double testTime = Double.parseDouble(testNode.getAttribute("time"));
 			NodeList resultNodes = testNode.getChildNodes();
+      int testTotal = 0;
+      List<Element> successNodes = new ArrayList<>();
```

Comment:
Looks like we only care about the number of successes, so this should be an `int` instead of a list.